packages/react-composites/src/composites/CallComposite/components/LobbyTile.tsx

```diff
@@ -70,6 +70,7 @@ export const LobbyTile = (props: LobbyTileProps): JSX.Element => {
               className={mergeStyles(titleStyle(palette, isVideoReady))}
               aria-live="assertive"
               data-ui-id="lobbyScreenTitle"
+              role="alert"
```

Comment:
We should add this to the Notice page also... but we can do that another time I suppose

Reply:
Oh I'll do that now!

Reply:
Done and tested!